sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/RsaCryptographyProvider.cs

```diff
@@ -222,13 +222,13 @@ public override UnwrapResult UnwrapKey(KeyWrapAlgorithm algorithm, byte[] encryp
 
         private byte[] Encrypt(byte[] data, RSAEncryptionPadding padding)
         {
-            using RSA rsa = KeyMaterial.ToRSA(true);
+            using RSA rsa = KeyMaterial.ToRSA();
             return rsa.Encrypt(data, padding);
         }
 
         private byte[] Decrypt(byte[] data, RSAEncryptionPadding padding)
         {
-            using RSA rsa = KeyMaterial.ToRSA();
+            using RSA rsa = KeyMaterial.ToRSA(true);
```

Comment:
Why did this flip?

Reply:
Mistakes were made previous. 🤦‍♂️

This shouldn't have been noticeable in production, though, because we wouldn't fetch a key with a private key for local operations. Apparently I didn't have the test coverage for this I thought I did, and it was newly-added test coverage that caught it.